cirq-core/cirq/protocols/json_serialization_test.py

```diff
@@ -237,7 +237,7 @@ def test_not_yet_serializable_no_superfluous(mod_spec: ModuleJsonTestSpec):
 
 
 @pytest.mark.parametrize('mod_spec', MODULE_TEST_SPECS, ids=repr)
-def test_mutually_exclusive_blacklist(mod_spec: ModuleJsonTestSpec):
+def test_mutually_exclusive_lists(mod_spec: ModuleJsonTestSpec):
```

Comment:
Optional: blocklist and passlist are self-explanatory alternatives.

Reply:
True, but I didn't think that these were necessary.  Should not serialize and not yet serialize I guess are block lists, but the term didn't really seem to fit.